vendor/modules.txt

```diff
@@ -25,9 +25,13 @@ github.com/google/go-cmp/cmp/internal/diff
 github.com/google/go-cmp/cmp/internal/flags
 github.com/google/go-cmp/cmp/internal/function
 github.com/google/go-cmp/cmp/internal/value
+# github.com/google/pprof v0.0.0-20230705174524-200ffdc848b8
+## explicit; go 1.19
```

Comment:
Any idea what this means - the reference to go 1.19 here and go 1.18 and 1.19 below?

Reply:
The file audo-updated by `go mod vendor`, which was required after cleaning it up with `go mod tidy`. It is a comment created by the tool, which I do not really care.

Reply:
It appears that it is more than a comment. They call it annotation? See https://github.com/golang/go/issues/37946. Other than that I was not able to  find the reference on how to read the `vendor/modules.txt` file.

Reply:
Ok, maybe the hash signs mean we're not really using those older versions.